packages/uui-menu-item/lib/uui-menu-item.element.ts

```diff
@@ -553,7 +554,7 @@ export class UUIMenuItemElement extends SelectOnlyMixin(
 
       #icon {
         display: inline-flex;
-        font-size: 16px;
+        font-size: 15.66px;
```

Comment:
It seems this would solve this instead.

```
width: 1.125em;
height: 1.125em;
```

which also solve some weird gap in TipTap toolbar buttons.

![image](https://github.com/user-attachments/assets/2862112b-c429-4e58-8248-1748a37569aa)

![image](https://github.com/user-attachments/assets/456c9791-39cc-4d98-ae12-77aebb358e2b)

Alternatively `1.125rem` instead, which seems to clear the gap between toolbar buttons, but affect a bit overall icon sizes.

![image](https://github.com/user-attachments/assets/5342815c-b58c-4154-b49f-b0600eb93eaf)

Reply:
Yes, that was also my first solution, but I realized I would end up changing the icons sizing across the whole backoffice :)

Reply:
I think is would be best to decide the icon sizes we want as the relative size depends on base font size.

Also it may fix the issue here, it would cause all other kind of issues throughout the backoffice.